charts/newrelic-infrastructure-v3/templates/controlplane/daemonset.yaml

```diff
@@ -11,10 +11,14 @@ metadata:
   annotations: {{ . | nindent 4 }}
   {{- end }}
 spec:
-  {{- if eq .Values.controlPlane.kind "DaemonSet"}}
+  {{- if eq .Values.controlPlane.kind "DaemonSet" }}
   {{- with .Values.updateStrategy }}
   updateStrategy: {{ toYaml . | nindent 4 }}
   {{- end }}
+  {{- else if eq .Values.controlPlane.kind "Deployment" }}
+  {{- with .Values.controlPlane.strategy }}
+  strategy: {{ toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
I didn't see that coming...

Reply:
https://www.youtube.com/watch?v=Ec4YbVP9R-A&t=46s

Reply:
Neither this.

Reply:
at some point we are going to have:
```
charts/newrelic-infrastructure-v3/templates/controlplane/daemonset.yaml
charts/newrelic-infrastructure-v3/templates/controlplane/deployment.yaml
```